storage/imx6ull-flash/flashdrv.c

```diff
@@ -720,6 +724,14 @@ int flashdrv_write(flashdrv_dma_t *dma, uint32_t paddr, void *data, char *aux)
 		*(flashdrv_common.bch + bch_flash0layout1) &= ~(0xffff << 16);
 		*(flashdrv_common.bch + bch_flash0layout1) |= flashdrv_common.rawmetasz << 16;
 	}
+	else if (aux == (char *)flashdrv_common.uncached_buf) {
```

Comment:
I would prefer to add explicit variable (eg. `skipMeta` or something similar) instead of condition based on pointer value

Reply:
Changed as suggested - I've added `skipMeta` variable.